src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,181 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final short GROWTH_FACTOR = 2;
```

Comment:
```suggestion
    private static final float LOAD_FACTOR = 0.75f;
```

Reply:
I know that default implementation has load factor, but I was about to reduce casting and so on, but sure I will correct it

Reply:
Done